### DISCUSSION TITLE
CXFTEOXK USB WiFi adapter works

### DISCUSSION THREAD

**Discussion by CaseySJ** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#298] [General]
Just reporting another working USB WiFi driver in Sonoma:

Make: CXFTEOXK
Model: B093FT3H7L 
Amazon Link: https://www.amazon.com/dp/B093FT3H7L

Sample result with Sonoma:
<img width="739" alt="Screenshot 2023-06-09 at 11 50 43 AM" src="https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/48335376/392d5a1b-a288-4850-92c3-13856de9ebd3">

**Comment by chris1111** [comment]
Thanks @CaseySJ  Done
